pkg/policyfilter/state.go

```diff
@@ -426,7 +430,6 @@ func (m *state) AddPolicy(polID PolicyID, namespace string, podLabelSelector *sl
 
 // DelPolicy will destroly all information for the provided policy
 func (m *state) DelPolicy(polID PolicyID) error {
-
```

Comment:
Please don't include whitespace changes in the patches that change functionality. Either skip them or add them in a  seperate patch.

Reply:
This comment was resolved, but there are still whitespaces in the patch.

Reply:
umm . my naughtly formatter